data/categories.txt

```diff
@@ -5297,7 +5299,7 @@ pl:3^Naprawa samochodu|warsztat samochodowy|warsztat|samochód|auto
 pt:^Oficina de reparação de automóveis|estação de serviço
 pt-BR:^Oficina de automóveis|mecânica|mecânico|auto elétro
 ro:^Atelier de reparații auto
-es:^Taller|3reparaciones auto|coche|automóvil|3auto|carro
+es:^Taller|3reparación de automóviles|coche|automóvil
```

Comment:
Is "Taller" really the most used term for this?

Reply:
Yes, it is